c/storage_mem.c

```diff
@@ -110,16 +114,18 @@ static void memStorageSetString(MemStorage *storage, const char *key, const char
   const char *valueCopy = memStorageCopyString(storage, value);
   if (!keyCopy || !valueCopy) {
     if (keyCopy) {
-      memStorageFree((char*)keyCopy);
+      memStorageFree((void*)keyCopy);
```

Comment:
That's the beauty of `void *`, you don't have to cast anymore.

Reply:
Still need the cast because of `const char*`. `void*` would help in case of `char *`.